python/pyspark/sql/functions.py

```diff
@@ -840,6 +841,20 @@ def size(col):
     return Column(sc._jvm.functions.size(_to_java_column(col)))
 
 
+@since(1.5)
+def sort_array(col):
+    """
+    Collection function: sorts the input array for the given column in ascending order.
```

Comment:
Hive does not support, maybe we can support it and be compatible with Hive (in ascending order by default)